samples/system-test/snippets_list_assets.test.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright 2019, Google, LLC.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+/**
+ * @fileoverview Smoke-test for snippets_list_assets.js
+ */
+
+const {assert} = require('chai');
+const organization_id = process.env['GCLOUD_ORGANIZATION'];
```

Comment:
I would really prefer to not require specialized env vars when running our tests.   We have a special test project we use for resources, but I don't know that we have a special test 'org'.  Where is a good place for us to run these tests?